geniza/corpus/models.py

```diff
@@ -220,6 +220,9 @@ def iiif_images(self):
                     # label provides library's recto/verso designation
                     labels.append(canvas.label)
             except IIIFException:
+                logger.warning(
+                    "IIIFException raised by from_url with URL: %s" % self.iiif_url
```

Comment:
this is the log message I would write — do you think it gives enough context?

```suggestion
                    "Error loading IIIF manifest: %s" % self.iiif_url
```

Reply:
Yes!